cross/libaom/PLIST

```diff
@@ -0,0 +1,15 @@
+bin:bin/aomdec
+bin:bin/aomenc
+lib:lib/libaom.so.0
+lnk:lib/libaom.so
+rsc:lib/pkgconfig/aom.pc
+rsc:include/aom/aom.h
+rsc:include/aom/aom_codec.h
+rsc:include/aom/aom_frame_buffer.h
+rsc:include/aom/aom_image.h
+rsc:include/aom/aom_integer.h
+rsc:include/aom/aom.h
+rsc:include/aom/aom_decoder.h
+rsc:include/aom/aomdx.h
+rsc:include/aom/aomcx.h
+rsc:include/aom/aom_encoder.h
```

Comment:
There is no need to include headers and pkgconfig files in generated package

Reply:
Indeed.  I had found an issue in my earlier builds where pkgconfig file was creating errors but I guess that was due to the version mismatch at the time.  I tried removing the headers & pkgconfig files from the `PLIST` and re-did a test build on selected arch successfully.  Changes now committed.